TheSadRogue.Primitives/TheSadRogue.Primitives.csproj

```
@@ -2,11 +2,11 @@
 
   <PropertyGroup>
 	<!-- Basic package info -->
-    <TargetFrameworks>netstandard2.1;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netcoreapp3.1;net5</TargetFrameworks>
```

This should be net5.0

```suggestion
    <TargetFrameworks>netstandard2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```

---

Er, whoops!  Fixed :D